file-formats/edot/type/zif_aff_edot_v1.intf.abap

```diff
@@ -0,0 +1,103 @@
+"! <p class="shorttext synchronized" lang="en">Create eDocument Types</p>
+"! eDocument types that can be used in eInvoice Process
+INTERFACE zif_aff_edot_v1 PUBLIC.
+
+  "! <p class="shorttext">eDocument Type</p>
+  "! Specify eDocument types
+  TYPES ty_edoc_type         TYPE c LENGTH 10.
+
+  "! <p class="shorttext">Description</p>
+  "! Description
+  TYPES ty_short_description TYPE c LENGTH 30.
+  TYPES:
+    "! <p class="shorttext">SRAL Configuration Type</p>
+    "! eDocument SRAL Configuration type
+    BEGIN OF ty_sral_configuration,
+      "! <p class="shorttext">File Type</p>
+      "! File Type
+      "! $required
+      file_type               TYPE c LENGTH 10,
+      "! <p class="shorttext">File Structure Type</p>
+      "! File Structure type
+      "! $required
+      file_structure_type     TYPE c LENGTH 30,
+      "! <p class="shorttext">File Description</p>
+      "! File Description type
+      file_description_type   TYPE c LENGTH 60,
+      "! <p class="shorttext">File Cloud Relevancy</p>
+      "! File Cloud Relevancy type
+      not_cloud_relevant_type TYPE abap_bool,
```

Comment:
I see for a lot of fields a mismatch in the title ("File Cloud Relevancy"; and its description) and the field name (`not_cloud_relevant_type`). It would be great if this can be harmonized in general.

However, in this case it seems to me just wrong:

File Cloud Relevancy = "Yes" vs. **not**_cloud_relevant_type = true